test/unit/auth/user-record.spec.ts

```diff
@@ -439,6 +584,12 @@ describe('MultiFactorSettings', () => {
       enrolledAt: now.toISOString(),
       phoneInfo: '+16505556789',
     }),
+    new TotpMultiFactorInfo({
+      mfaEnrollmentId: 'enrollmentId5',
```

Comment:
why 'enrollmentId5' not 'enrollmentId3'?

Reply:
Because there are 2 invalid configs in the `mfaInfo` list called `enrollmentId3` and `enrollmentId4`.
But seems like we can regroup or write better variable names for those.